### PR TITLE
perfect document

### PR DESCRIPTION
**Pull request by tanjunchen**
### Ⅰ. Describe what this PR did
perfect document
### Ⅱ. Does this pull request fix one issue?
None
### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
No Need

### CODE REVIEW

File: cmd/dfdaemon/app/init.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func initDfdaemon(cfg config.Properties) error {
 	return nil
 }
 
-// initLogger initialize the global logrus logger
+// initLogger initializes the global logrus logger
```
Comment (yeya24) [inline cmd/dfdaemon/app/init.go]: Why do you don't add . here?

File: cmd/dfdaemon/app/init.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ func initLogger(cfg config.Properties) error {
 	return errors.Wrap(dflog.Init(logrus.StandardLogger(), opts...), "init log")
 }
 
-// cleanLocalRepo checks the files at local periodically, and delete the file when
+// cleanLocalRepo checks the files at local periodically, and deletes the file when
```
Comment (yeya24) [inline cmd/dfdaemon/app/init.go]: ditto

File: cmd/dfget/app/root.go [hunk 3]
```diff
@@ -151,7 +151,7 @@ func initClientLog() error {
 		dflog.WithDebug(cfg.Verbose),
 	}
 
-	// once cfg.Console is set, process should also output log to console
+	// once cfg.Console is setted, process should also output log to console
```
Comment (yeya24) [inline cmd/dfget/app/root.go]: once -> Once . also .     setted -> set

File: cmd/dfget/app/server.go [hunk 4]
```diff
@@ -88,7 +88,7 @@ func initServerLog() error {
 		dflog.WithDebug(cfg.Verbose),
 	}
 
-	// once cfg.Console is set, process should also output log to console
+	// once cfg.Console is setted, process should also output log to console
```
Comment (yeya24) [inline cmd/dfget/app/server.go]: Once  set .

File: cmd/dfdaemon/app/root.go [hunk 5]
```diff
@@ -164,7 +164,7 @@ func getConfigFromViper(v *viper.Viper) (*config.Properties, error) {
 }
 
 // decodeWithYAML returns a mapstructure.DecodeHookFunc to decode the given
-// types by unmarshalling from yaml text
+// types by unmarshalling from yaml text.
```
Comment (yeya24) [inline cmd/dfdaemon/app/root.go]: If you add . here, you should use uppercase Types

File: cmd/dfdaemon/app/init.go (hunk 1 above)
Comment (tanjunchen) [reply cmd/dfdaemon/app/init.go]: image

Comment (tanjunchen) [reply cmd/dfdaemon/app/init.go]: // initLogger initializes the global logrus logger
func initLogger(cfg config.Properties) error {
	current, err := user.Current()
	if err != nil {
		return errors.Wrap(err, "get current user")
	}

	logFilePath := filepath.Join(current.HomeDir, ".small-dragonfly/logs/dfdaemon.log")

	opts := []dflog.Option{
		dflog.WithLogFile(logFilePath),
		dflog.WithSign(fmt.Sprintf("%d", os.Getpid())),
		dflog.WithDebug(cfg.Verbose),
		dflog.WithConsole(),
	}

	logrus.Debugf("use log file %s", logFilePath)

	return errors.Wrap(dflog.Init(logrus.StandardLogger(), opts...), "init log")
}

Comment (tanjunchen) [reply cmd/dfdaemon/app/init.go]: Function

File: cmd/dfdaemon/app/init.go (hunk 2 above)
Comment (tanjunchen) [reply cmd/dfdaemon/app/init.go]: // cleanLocalRepo checks the files at local periodically, and deletes the file when
// it comes to a certain age(counted by the last access time).
// TODO: what happens if the disk usage comes to high level?

Does it really need to be added . after when?

File: cmd/dfdaemon/app/root.go (hunk 5 above)
Comment (tanjunchen) [reply cmd/dfdaemon/app/root.go]: // decodeWithYAML returns a mapstructure.DecodeHookFunc to decode the given
// types by unmarshalling from yaml text.   

Isn't this a complete statement?
Really  remove .  ?

**Comment by tanjunchen** [comment]
@yeya24 Are there any other questions?  Thanks!!!

File: cmd/dfdaemon/app/root.go (hunk 5 above)
Comment (yeya24) [reply cmd/dfdaemon/app/root.go]: I am sorry I missed there is one more line above

File: cmd/dfdaemon/app/init.go (hunk 2 above)
Comment (yeya24) [reply cmd/dfdaemon/app/init.go]: I am sorry I missed there is one more line above

File: cmd/dfdaemon/app/root.go (hunk 5 above)
Comment (Starnop) [reply cmd/dfdaemon/app/root.go]: Haha, maybe you need to have a good rest.  Pay attention to your body. 😄

**Comment by Starnop** [comment]
LGTM.

File: cmd/dfdaemon/app/root.go (hunk 5 above)
Comment (tanjunchen) [reply cmd/dfdaemon/app/root.go]: hahaha

**Pull request merged**